.eslintrc.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
```

Comment:
It would be nice to set a rule for the maximum length allowed per line in the eslint file.